plugins/mcreator-core/datalists/defaultfeatures.yaml

```diff
@@ -12,6 +14,7 @@
 - JungleGrass
 - JungleTreeForest
 - Lakes
+- LushCavesDecoration

```

Comment:
I couldn't find this one in the MC source. Perhaps you meant `LushCavesVegetationFeatures` or `LushCavesSpecialOres`?

Reply:
It's Yarn's name, but it's the `LushCavesVegetationFeatures`.